lib/tendrl/flow.rb

```diff
@@ -0,0 +1,121 @@
+module Tendrl
+  class Flow
+
+    METHOD_MAPPING = { 'create' => 'POST', 'update' => 'PUT', 'delete' =>
```

Comment:
While such convention based approach is fine, it needs to be standardised and made part of the common library across the project.